src/sources/datadog_agent/mod.rs

```diff
@@ -443,8 +443,8 @@ pub(crate) async fn handle_request(
             } else {
                 out.send_batch(events).await
             }
-            .map_err(move |error: crate::source_sender::ClosedError| {
```

Comment:
I _think_ it's ok to drop these `move`'s but please double check! 🙇

Reply:
Yep, it look okay to drop `move` because the only variable captured by the closure is `count`, which is not captured by reference.